internal/server/conf.go

```diff
@@ -64,6 +64,8 @@ type Conf struct {
 	LogRequestPayloads bool `yaml:"logRequestPayloads" conf:",example=false"`
 	// PlaygroundEnabled defines whether the playground API is enabled.
 	PlaygroundEnabled bool `yaml:"playgroundEnabled" conf:",ignore"`
+	// DisableAPIExplorer disables the API explorer UI from being served on the HTTP port.
+	DisableAPIExplorer bool `yaml:"disableAPIExplorer" conf:",example=false"`
```

Comment:
We have `*Enabled` or `*Disabled` for other bools so maybe this should be `APIExplorerEnabled` with a default of `true` to match `MetricsEnabled`?